docs/USER-DOCUMENTATION.md

```diff
@@ -149,6 +149,14 @@ In Windows:
 set ETCHER_DISABLE_UPDATES=1
 ```
 
+Simulate an update alert
+------------------------
+
+You can set the `ETCHER_FAKE_S3_LATEST_VERSION` environment variable to a valid
+semver version (greater than the current version) to trick the application into
+thinking that what you put there is the latest available version, therefore
+causing the update notification dialog to be presented at startup.
```

Comment:
Not sure what would be the point of describing that there, given that `ETCHER_FAKE_S3_LATEST_VERSION` fakes the latest remote version independently on the release type. For example, the current version will be compared against `ETCHER_FAKE_S3_LATEST_VERSION`, independently on if its a production, snapshot, or unknown release type.

> if Etcher detects that it's a snapshot build, I wonder if it'd make sense to filter out any non-snapshot versions that it gets from the version-update method? (and vice-versa for production)

I think that's a great idea. I'll address that in this PR.